build.gradle.kts

```diff
@@ -249,8 +249,6 @@ tasks.register("javaPreCommit") {
   dependsOn(":beam-validate-runner:build")
   dependsOn(":examples:java:build")
   dependsOn(":examples:java:preCommit")
-  dependsOn(":examples:java:sql:build")
```

Comment:
This separate module was intended: https://github.com/apache/beam/pull/34322#discussion_r2023215098

I try to avoid sql dependencies in main example. Any chance make the CI work with separate module?

Reply:
Done here: #35183